packages/store-ui/src/Toast/index.tsx

```diff
@@ -4,27 +4,60 @@ import { Box } from 'theme-ui'
 import type { ReactNode, FC } from 'react'
 
 import { CrossIcon } from './CrossIcon'
+import { ErrorIcon } from './ErrorIcon'
+import { WarningIcon } from './WarningIcon'
+import { CheckIcon } from './CheckIcon'
+
+type ToastType = 'warning' | 'error' | 'success'
+
+const typeIcons: Record<ToastType, FC> = {
+  error: ErrorIcon,
+  warning: WarningIcon,
+  success: CheckIcon,
+}
 
 interface Props {
   message: ReactNode
   isVisible: boolean
   hideToast: () => void
+  type: ToastType
+  disablePortal?: boolean
```

Comment:
What is the use case?

Reply:
Inside the store, take a look here => https://github.com/vtex-sites/storecomponents.store/pull/730/files#diff-81402ac32267f1ab0ac2d93277358029a19cae82dbdf9668f8d9f9dbfb3d222dR5-R11

Reply:
What does it do? Maybe the name of the prop should be the UI effect instead of the implementation detail

Reply:
Yeah, i removed the disablePortal prop